rm/rm-node/src/main/java/org/ow2/proactive/resourcemanager/utils/RRDSigarDataStore.java

```diff
@@ -190,7 +190,10 @@ void sample(RrdDb dataBase, long timeInMs) throws IOException {
                     logger.trace("Cannot read attribute " + attrName + " for object " + objectName + ": " +
                                  e.getMessage());
                 }
-
+            } catch (UnsupportedOperationException e) {
+                if (logger.isTraceEnabled()) {
```

Comment:
I would do something different, if logger.isDebugEnabled() then print this message (in warning)
if logger.isTraceEnabled() then print the full exception